user_docs/en/userGuide.md

```diff
@@ -2215,34 +2190,59 @@ However, in order for you to read the context (i.e. that you are in a list and t
 
 To toggle focus context presentation from anywhere, please assign a custom gesture using the [Input Gestures dialog](#InputGestures).
 
-##### Interrupt speech while scrolling {#BrailleSettingsInterruptSpeech}
+##### Show selection {#BrailleSettingsShowSelection}
 
 | . {.hideHeaderRow} |.|
 |---|---|
 |Options |Default (Enabled), Enabled, Disabled|
 |Default |Enabled|
 
-This setting determines if speech should be interrupted when the Braille display is scrolled backwards/forwards.
-Previous/next line commands always interrupt speech.
+This setting determines if selection indicator (dots 7 and 8) is shown by the braille display.
+The option is enabled by default so the selection indicator is shown.
+The selection indicator might be a distraction while reading.
+Disabling this option may improve readability.
 
-On-going speech might be a distraction while reading Braille.
-For this reason the option is enabled by default, interrupting speech when scrolling braille.
+To toggle show selection from anywhere, please assign a custom gesture using the [Input Gestures dialog](#InputGestures).
 
-Disabling this option allows speech to be heard while simultaneously reading Braille.
+##### Avoid splitting words when possible {#BrailleSettingsWordWrap}
 
-##### Show selection {#BrailleSettingsShowSelection}
+If this is enabled, a word which is too large to fit at the end of the braille display will not be split.
+Instead, there will be some blank space at the end of the display.
+When you scroll the display, you will be able to read the entire word.
+This is sometimes called "word wrap".
+Note that if the word is too large to fit on the display even by itself, the word must still be split.
+
+If this is disabled, as much of the word as possible will be displayed, but the rest will be cut off.
+When you scroll the display, you will then be able to read the rest of the word.
+
+Enabling this may allow for more fluent reading, but generally requires you to scroll the display more.
+
+##### Unicode normalization {#BrailleUnicodeNormalization}
+| . {.hideHeaderRow} |.|
+|---|---|
+|Options |Default (Disabled), Enabled, Disabled|
+|Default |Disabled|
```

Comment:
could you please move the tables to the end of each section? this is the new preferred format as documented in [userGuideStandards](https://github.com/nvaccess/nvda/blob/master/projectDocs/dev/userGuideStandards.md)